OpenNefia.Content/UI/Element/FeatWindow.cs

```diff
@@ -0,0 +1,311 @@
+﻿using Love;
+using OpenNefia.Content.Inventory;
+using OpenNefia.Content.Prototypes;
+using OpenNefia.Content.UI.Element.List;
+using OpenNefia.Core.Audio;
+using OpenNefia.Core.Input;
+using OpenNefia.Core.IoC;
+using OpenNefia.Core.Locale;
+using OpenNefia.Core.Maths;
+using OpenNefia.Core.Prototypes;
+using OpenNefia.Core.Rendering;
+using OpenNefia.Core.UI.Element;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.UI.Element
+{
+    public class FeatWindow : UiWindow
+    {
+        public abstract record FeatData
+        {
+            public virtual string Name { get; } = "";
+            public virtual string Description { get; } = "";
+            public virtual Core.Maths.Color Color { get; } = Core.Maths.Color.Black;
```

Comment:
Nitpick: import `Core.Maths`?

Reply:
it is imported, there's an ambiguity between that and Love.Color